src/controllers/accounts.controller.js

```diff
@@ -92,19 +83,55 @@ exports.register = async (req, res, next) => {
           item[key] = req.body[key];
           break;
         case 'referral':
-          continue; // TODO update when referral code has been defined
+          if (req.body[key].length != 6) {
```

Comment:
the referral code is not required, is it? 
only when the user's email doesn't belong to some organization, the referral code is necessary.

Reply:
This is optional; referral code logic is in switch statement